ios/Application/DependencyInjection/Sources/DependencyInjection/KMPDependency.swift

```diff
@@ -27,7 +28,10 @@ final class KMPKoinDependency: KMPDependency {
             Logger.app.info("Koin Started")
         }
         
-        let koinApplication = KoinIOSKt.doInitKoinIos(doOnStartup: onStartup)
+        let koinApplication = KoinIOSKt.doInitKoinIos(
+            doOnStartup: onStartup,
+            analyticsSource: Container.shared.analyticsProvider()
```

Comment:
Can we rename the parameter to `analyticsProvider` to unify it?

Reply:
True.. We use `Provider` naming in KMP for shared providers with implementations on platforms (in Heureka, for example), so that's one way to solve it 😅